### DISCUSSION TITLE
Download button on main page not working correctly

### DISCUSSION THREAD

**Discussion by Yentzen** [mitmproxy/mitmproxy#5446] [General]
On the mitmproxy.org website, the red button with caption "Download Windows Installer" is redirecting to a non-existing executable. It is missing "-x64" in the executable name. 
It is currently pointing to https://snapshots.mitmproxy.org/8.1.1/mitmproxy-8.1.1-windows-installer.exe while the correct executable patch is https://snapshots.mitmproxy.org/8.1.1/mitmproxy-8.1.1-windows-x64-installer.exe
![image](https://user-images.githubusercontent.com/8031084/177628867-36300e69-4cf3-4da6-aabb-0cd84fab3b2b.png)

**Comment by mhils** [comment]
D'oh! Thanks for the heads-up, fixed! 😃 🍰
